content/operate/rs/installing-upgrading/install/install-on-linux.md

```diff
@@ -29,37 +29,57 @@ To install Redis Enterprise Software, use the command line:
 1.  _(Optional)_ Use the {{< download "GPG key file" "../GPG-KEY-redislabs-packages.gpg" >}} to confirm the authenticity of Ubuntu/Debian or RHEL RPM packages:
 
     - For Ubuntu:
-        1. Import the key:  
+        1. Import the key:
         ```sh
         gpg --import <path to GPG key>
-        ```  
-        2. Verify the package signature: 
-        ```sh 
+        ```
+        2. Verify the package signature:
+        ```sh
         dpkg-sig --verify </path-to/package.deb>
         ```
 
     - For RHEL:
-        1. Import the key:  
+        1. Import the key:
         ```sh
         rpm --import <path to GPG key>
         ```
-        2. Verify the package signature:  
+        2. Verify the package signature:
          ```sh
          rpm --checksig </path-to/package.rpm>
          ```
 
-1. To start the installation process, run the installation script. See [installation script options]({{< relref "/operate/rs/installing-upgrading/install/install-script" >}}) for a list of command-line options you can add to the following command:
+1. To start the installation process, run the installation script:
 
+    **For standard installation:**
     ```sh
     sudo ./install.sh
     ```
 
+    **For custom installation directories:**
+
+    If you need to specify custom storage paths for persistent or ephemeral storage during cluster setup, you must install Redis Enterprise Software to custom directories. See [installation script options]({{< relref "/operate/rs/installing-upgrading/install/install-script" >}}) for a complete list of command-line options.
```

Comment:
If you add the install script options back to the top-level step, we can remove it here. We also already provide a link to the custom install instructions immediately below the example.
```suggestion
    If you need to specify custom storage paths for persistent or ephemeral storage during cluster setup, you must install Redis Enterprise Software to custom directories.
```